toolkit/tools/pkg/imagecustomizerlib/customizeverity.go

```diff
@@ -91,6 +96,47 @@ func buildDracutModule(dracutModuleName string, dracutDriverName string, imageCh
 	return nil
 }
 
+func updateFstab(buildDir string, imageChroot *safechroot.Chroot, imageConnection *ImageConnection) error {
+	var err error
+
+	diskPartitions, err := diskutils.GetDiskPartitions(imageConnection.Loopback().DevicePath())
+	if err != nil {
+		return err
+	}
+
+	rootfsPartition, err := findRootfsPartition(buildDir, diskPartitions)
+	if err != nil {
+		return err
+	}
+
+	fstabFile := filepath.Join(imageChroot.RootDir(), "etc", "fstab")
+	lines, err := file.ReadLines(fstabFile)
```

Comment:
You can use [diskutils.ReadFstabFile()](https://github.com/microsoft/azurelinux/blob/3.0-dev/toolkit/tools/imagegen/diskutils/fstab.go#L130) to fully parse the fstab file. Using this might make it easier to robustly modify the fstab file.

Note: You may or may not want to add `options` to the `findmnt`'s `--output` list. The `options` column gives you the full options field. As opposed to `vfs-options` and `fs-options` which splits up the `option` value.

Reply:
Once I get the `[]diskutils.FstabEntry`, how be able to modify the fstab file?

Reply:
Once you've finished editing the fstab file, you can rewrite the fstab file from scratch.

Reply:
Fixed.